### ISSUE TITLE
Accessibility: extra letter being spoken on subscriptions list

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#326]
**Description**  
Talkback reads an extra letter before the name and instance. Some examples are "C Chat chat · lemmy.one", "C Chat chat · beehaw.org", and "C Cryptocurrency cryptocurrency · lemmy.ml" instead of "Chat chat · lemmy.one", "Chat chat · beehaw.org", or "Cryptocurrency cryptocurrency · lemmy.ml".

**How to Reproduce**  
1. Go to Settings > Accessibility > Talkback and turn it on
2. Open Thunder
3. Open the side menu and tap subscriptions in your list

**Expected Behavior**  
Talkback should read (CommunityTitle, CommunityName, Instance) as it used to.

**Screenshots**  

**Device & App Version:**  
 - Device: [e.g. iPhone6] OnePlus Nord N200
 - OS: [e.g. iOS8.1] Android 13
 - Version [e.g. 22] Thunder v0.2.1-alpha.2+12

**Additional Context**  
I *think* this may be caused by the addition of the community icons as it only started after updating from 0.2.1+11. My guess is that instead of reading "unlabeled" or "image" its reading the first letter of the community title. If so the image element can be totally skipped by the screen reader as its not important.

**Comment by hjiangsu** [comment]
That seems about right - I think the issue lies in the screen reader reading the first letter of a community. Does this happen when there is a community icon or just when there's no community icon?

**Comment by shortwavesurfer2009** [comment]
Both communities with or without. Communities without have a letter instead of a picture

**Comment by shortwavesurfer2009** [comment]
Which release? V0.2.1-alpha.3+12 still does this

**Comment by hjiangsu** [comment]
Oh shoot - I think I fixed a different issue with the search part

In the search page, when you search for a community, it was also doing the same thing and I thought that was what this ticket was about (oops)

**Comment by shortwavesurfer2009** [comment]
> 

Total fail. No worries, it happens
